SimG4CMS/HGCalTestBeam/test/HGCalTBCERN170_cfg.py

```diff
@@ -95,16 +95,10 @@
 process.VtxSmeared.MaxY =  7.5
 process.g4SimHits.HGCSD.RejectMouseBite = True
 process.g4SimHits.HGCSD.RotatedWafer    = True
-process.g4SimHits.OnlySDs = ['HGCalSensitiveDetector', 
```

Comment:
Out of curiosity, which is the difference between 'HGCalSensitiveDetector' and 'HGCSensitiveDetector'?